controllers/postController.js

```diff
@@ -3,21 +3,22 @@ const { isValidString, isValidNumber } = require("../utils/validation");
 
 const upsert = async (post) => {
```

Comment:
```suggestion
const upsert = async (req) => {
```
controller 함수의 파라미터 이름을 `req`, `res`, `next`으로 통일하면 어떨까요?
공식 문서를 참고해서, 콜백 함수 파라미터 이름을 이렇게 정하면 `keywordController`와도 맞출 수 있을 것 같습니다.
- [Express Routing](https://expressjs.com/en/guide/routing.html)
- 발췌 `keywordController`
https://github.com/Team-Bloblow/Bloblow-Server/blob/70491a15b346b1a5ed7f08eae834f21797f41940/controllers/keywordController.js#L6-L9

Reply:
저는 해당 로직을 클라이언트쪽에서 바로 받는 것이 아니라 전달받은 데이터에 대해 매개변수명을 지었는데
생각해보니 다른 로직에서도 해당 함수를 사용할 수 있으며, 파일 내 매개변수를 통일시키는 것이 더 좋을 것 같네요! 
제안해주신 대로 반영했습니다!